alerts/machine-config-operator/MachineConfigControllerDrainError.md

```diff
@@ -0,0 +1,114 @@
+# MCCDrainError
+
+## Meaning 
+
+Alerts the user to when the Machine Config Operator 
+failed to drain a node. Preventing the MCO from restarting 
+the node. 
+
+>Alerts the user to a failed node drain. Always triggers when the failure
+>happens one or more times.
+
+This alert will fire as a warning when: 
+
+- If the mco is unable to drain the node for an hour
+- When the failure occurs one or more times in a node 
+
+## Impact 
+
+If the mco fails to drain a node it will be unable to reboot the node. 
+Preventing any changes via MachineConfig or a cluster upgrade to be applied. 
```

Comment:
```suggestion
If the MCO fails to drain a node, it will be unable to reboot the node, which prevents any changes to the cluster through an machine config and prevents a cluster upgrade. 
```

Reply:
Agreed, Applied the change